apis/metal3.io/v1alpha1/baremetalhost_types_test.go

```diff
@@ -617,51 +617,56 @@ func TestSetHardwareProfile(t *testing.T) {
 }
 
 func TestHasBMCDetails(t *testing.T) {
+}
```

Comment:
I'm wondering why you removed the TestBMCDetails function, perhaps it is no longer needed?